### ISSUE TITLE
Documentation should be updated showing how to add custom theme(s)

### ISSUE THREAD

**Issue by IvanDrag0** [Badgerati/Pode.Web#502]
The current documentation on the website revolves around the current default theme. The documentation should be updated to show how to create custom theme(s) and/or how to modify the current theme (modify the .pode template files).

**Comment by Badgerati** [comment]
Hi @IvanDrag0,

Documentation for this for v1 will exist, as I'm working on making it easier to make custom themes and custom elements.

It won't contain anything relating to editing the inbuilt .pode template files though, as this is unsupported.

**Comment by IvanDrag0** [comment]
Hi @Badgerati ,

Is the work currently in GitHub? I'd like to take a look at it and see if I could help the development/testing part :)

**Comment by Badgerati** [comment]
Yes, all work for v1 is currently in develop :) I've moved the rendering of elements to javascript, which can be found here https://github.com/Badgerati/Pode.Web/blob/develop/src/Templates/Public/scripts/templates.js

I've yet to document how creating custom elements work, as the code needs a little clean-up; but in it's current format it's possible by extending the element classes/functions - or writing your own.
